python/python/tests/test_search.py

```diff
@@ -0,0 +1,176 @@
+# Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+
+from typing import List
+
+import pytest
+from glide.async_commands.core import InfoSection
+from glide.async_commands.redis_modules.search.commands import Search
+from glide.async_commands.redis_modules.search.field import (
+    CoordinateSystem,
+    Field,
+    GeoField,
+    GeoShapeField,
+    NumericField,
+    TagField,
+    TextField,
+    VectorAlgorithm,
+    VectorField,
+)
+from glide.async_commands.redis_modules.search.index import Index, IndexType
+from glide.constants import OK
+from glide.exceptions import RequestError
+from glide.redis_client import TRedisClient
+from tests.test_async_client import (
+    check_if_server_version_lt,
+    get_random_string,
+    parse_info_response,
+)
+
+
+@pytest.mark.asyncio
+class TestSearchModule:
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    async def test_search_module_loaded(self, redis_client: TRedisClient):
+        res = parse_info_response(await redis_client.info([InfoSection.MODULES]))
+        assert "search" in res["module"]
+
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    async def test_json_module_loadded(self, redis_client: TRedisClient):
```

Comment:
why are you testing the JSON module in the search module tests?
also - loadded -> loaded

Reply:
In order to create json indexes, json module needs to be loaded, maybe it would be better to combine both tests into a single test

Reply:
I don't think we should combine the modules tests, and I think that this test should belong to json tests only. You can have an assertion in the json indexes tests to verify that json module was loaded